dashboards/rabbitmq/metadata.yaml

```
@@ -9,3 +9,12 @@ sample_dashboards:
     related_integrations:
       - id: rabbitmq
         platform: GCE
+  -
+    category: RabbitMQ
+    id: rabbitmq-prometheus-overview
```

The id here should match the file name.

```suggestion
    id: rabbitmq-prometheus
```